examples/units/units_sample.py

```diff
@@ -19,14 +19,14 @@
 
 cms = cm * np.arange(0, 10, 2)
 
-fig, axs = plt.subplots(2, 2)
+fig, axs = plt.subplots(2, 2, tight_layout=True)
```

Comment:
will labels show w/ constrained_layout?

Reply:
Yes we should spell this layout='constrained' if possible.

Reply:
Thanks for catching, should be fixed now.